FSL-DRAFT-Apache-2.0.template.md

```diff
@@ -4,19 +4,19 @@
 
 FSL-DRAFT-Apache-2.0
 
-## TERMS AND CONDITIONS
-
-### Notice
+## Notice
```

Comment:
MIT refers to the "above copyright notice," so this section should survive even when the Terms and Conditions terminate.